resources/eventing/charts/publisher-proxy/values.yaml

```diff
@@ -18,10 +18,11 @@ nats:
 metrics:
   config:
     port: 9090
-    portName: http-metrics
+    portName: "-metrics"
     nameSuffix: "-metrics"
```

Comment:
for reviewer:

We could potentially merge those two constants (`portName` and `nameSuffix`), as they are equal. But I didn't do this, because for me those still refer to different thins: one is name of the port and the second one is used in metadata. So I would keep them separated in case want want to change one of them in the future.